app/views/ctc/questions/dependents/does_not_qualify_ctc/edit.html.erb

```diff
@@ -1,43 +1,42 @@
-<% @main_question = t("views.ctc.questions.dependents.does_not_qualify_ctc.title", name:  @dependent.first_name.upcase_first) %>
+<% @puerto_rico_filing = current_intake.puerto_rico_filing? %>
```

Comment:
If we're setting the variable purely for this template, I'd rather it not be an `@` variable.

Another idea is to adjust the controller to set this variable.

I'd prefer either of those rather than this approach where the `@` variable is created in the `erb`.

Not a blocker.